src/components/footer.js

```diff
@@ -1,10 +1,11 @@
 import React from "react"
+import Img from "gatsby-image"
 import PropTypes from "prop-types"
 import styled from "styled-components"
```

Comment:
So the import ordering would start after styled?

Reply:
Forgot to explain this, but the idea was that React imports always come first, then after that it's alphabetical order.